src/utils/createMusicMetadata.ts

```diff
@@ -61,35 +63,33 @@ const music_metadata = {
 	},
 }
 
-const createMusicMetadata = data => {
-	music_metadata.name = data.name
-	music_metadata.title = data.name
-	music_metadata.attributes.project = data.name
-	music_metadata.tags = [data.name, data.address]
-	music_metadata.artist = data.address
-	music_metadata.credits = [{ name: data.address, collaboratorType: 'creator' }]
-	music_metadata.image = data.image
-	music_metadata.artwork.uri = data.image
-	music_metadata.animation_url = data.song
-	music_metadata.losslessAudio = data.song
-	music_metadata.description = data.description
-	const project = {
-		title: data.name,
+const createMusicMetadata = ({artist, name, description, song, image}: FormData) => {
+	const metadata = {
+		name: `${artist} - ${name}`,
+		description,
+		mimeType: song.type,
+		image,
+		losslessAudio: song,
+		animation_url: song,
 		artwork: {
-			uri: data.image,
-			mimeType: '',
-			nft: '',
+			uri: image,
+			mimeType: image.type
+		},
+		attributes: {
+			artist
 		},
-		description: data.description,
-		type: '',
-		originalReleaseDate: '',
-		recordLabel: '',
-		publisher: '',
-		upc: '',
+		project: {
+			title: name,
+			artwork: {
+				uri: image,
+				mimeType: image.type
+			},
+			description
+		}
 	}
-	music_metadata.project = project
+	console.log('CREATING MUSIC METADATA', metadata)
 
-	return music_metadata
+	return metadata
```

Comment:
this is the most important file in the FE repo I disagree with these changes on a fundamental level.

Reply:
Catalog uses `artist name - song title` as the name for the NFT is this the same approach we want for songs?